src/QGCQFileDialog.h

```diff
@@ -40,7 +41,7 @@
 */
 
 class QGCQFileDialog : public QFileDialog {
-    
+    Q_DECLARE_TR_FUNCTIONS(QGCQFileDialog)
```

Comment:
Ah, had never heard of that before. I wonder where else this is a problem. Those may be harder to find. Does the Qt stuff spit out and error for that somehow?